src/Apps/Order2/Routes/Checkout/Components/PaymentStep/Order2PaymentForm.tsx

```diff
@@ -519,41 +464,33 @@ const PaymentFormContent: React.FC<PaymentFormContentProps> = ({
         return
       }
 
-      if (selectedPaymentMethod === "stripe-card") {
-        try {
-          const updateOrderPaymentMethodResult =
-            await setPaymentMutation.submitMutation({
-              variables: {
-                input: {
-                  id: order.internalID,
-                  paymentMethod: "CREDIT_CARD",
-                  stripeConfirmationToken: confirmationToken.id,
-                },
+      const paymentMethod = {
+        "stripe-card": "CREDIT_CARD",
+        "stripe-ach": "US_BANK_ACCOUNT",
+        "stripe-sepa": "SEPA_DEBIT",
+      }[selectedPaymentMethod]
```

Comment:
Noticing we have to map these strings in this component. Will think about how to refactor and improve it. For now, doing a simple mapping locally.